hibernate-core/src/main/java/org/hibernate/boot/internal/InFlightMetadataCollectorImpl.java

```diff
@@ -1664,7 +1664,7 @@ private void processValueResolvers(MetadataBuildingContext buildingContext) {
 
 		while ( ! valueResolvers.isEmpty() ) {
 			final boolean anyRemoved = valueResolvers.removeIf(
-					resolver -> resolver.apply( buildingContext )
+					resolver -> resolver == null || resolver.apply( buildingContext )
```

Comment:
I'm curious where `null` is being put into this queue?  That seems wrong

Reply:
You are right. Since the PR is quite old already, I still had that in as a workaround for a previously issue, but it's working fine now even without this. Thanks!